src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -1,15 +1,28 @@
 package core.mate.academy.model;
 
-/**
- * Add some custom fields that could be only in Bulldozer
- * Do not remove no-args constructor
- */
 public class Bulldozer extends Machine {
+    private String dumpType;
+
     public Bulldozer() {
     }
 
+    public Bulldozer(String name,String color,String dumpType) {
+        super();
+        this.setName(name);
+        this.setColor(color);
```

Comment:
it's better to add a constructor to the Machine class and inherit it here like that 
```suggestion
        super(name, color);
```